redfish-core/include/generated/enums/port.hpp

```diff
@@ -0,0 +1,237 @@
+#pragma once
```

Comment:
Why do we need this file?

Reply:
So sorry, I forgot to click the `submit` button yesterday :(